Makefile

```diff
@@ -33,17 +33,17 @@ deploy-shell: login-to-ecr
 
 .PHONY: build
 build:
-	$(MAKE) -C serverless-functions layer-codebuild STAGE=$(STAGE)
+	docker-compose run tester make -C serverless-functions layer-codebuild STAGE=$(STAGE)
 
 .PHONY: release
 release:
-	$(MAKE) -C serverless-functions deploy-change-service STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-ldap-publisher STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-person-api STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-notifications STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-curator STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-hris-publisher STAGE=$(STAGE)
-	$(MAKE) -C serverless-functions deploy-auth0-publisher STAGE=$(STAGE)
+	docker-compose run tester make -C serverless-functions deploy-change-service STAGE=$(STAGE)
```

Comment:
nit: we probably should just have a general deploy target in `serverless-functions/Makefile` so theres less to update